packages/react-native/template/_gitignore

```diff
@@ -64,3 +64,11 @@ yarn-error.log
 
 # testing
 /coverage
+
+# Yarn
+.yarn/*
```

Comment:
Which version of Yarn is creating those files? Asking as I never saw them

Reply:
Yarn v3, the one that is set by default in CLI starting from next version.